bug/unable register device

JGiter opened the pull request:
Fixed issue: User belonging to an approved organization cannot register a device with reason that he do not belong to an approved organization

kosecki123 commented:
This good change, would it be possible to remove `IUserWithRealtionsId` ? Also rename IUserWithRelations` to `IUser`? I don't see any purpose of having 2 different interfaces based on that PR, maybe they used somewhere else though.

Commit:
chore(origin-backend-core): use IOrganization in IUser

BREAKING CHANGE: IUser.organization changed from number to full organization object

Commit:
chore(origin-backend): replace IUserWithRelationIds with IUser

BREAKING CHANGE: findOne returns user with full organization object

Commit:
chore(origin-backend-client): replace IUserWithRelationIds with IUser

BREAKING CHANGE: IUser have full org object

Commit:
chore(exchange): replace IUserWithRelationIds with IUser

Commit:
chore(origin-backend): replace IUserWithRelationIds with IUser

Commit:
chore(origin-backend-utils): replace IUserWithRelationIds with IUser

Commit:
chore(origin-ui-core): use organization from user

Commit:
chore(origin-ui-core): replace IUserWithRelationIds with IUser
